docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -240,64 +293,30 @@ Ensure your installation of [Camunda 8 Run](/self-managed/setup/deploy/local/c8r
-:::
-
-</TabItem>
-<TabItem value="sm">
+Next, check for a successful start in Operate.
 
 1. Open Operate at `http://localhost:8080/operate`, and select **Processes** from the top bar.
 2. In the **Process** panel, use the **Name** drop-down to select your process.
 3. A visualization of your running process instance now displays in Operate, and your user task is marked with a green **token** icon. This means that a task is waiting to be worked on in Tasklist.
 
-   <img src={OperateHumanTasks} alt="Process instance monitoring in Operate" />
-
-</TabItem>
-</Tabs>
+   <img src={OperateHumanTasks} alt="Process instance monitoring in Operate." />
 
-## Step 5: Complete a user task
+Finally, check that you can complete a user task in Tasklist.
 
-When the process instance arrives at the user task, a new user task instance is created at Zeebe. The process instance stops at this point and waits until the user task is completed. Applications like [Tasklist](/components/tasklist/introduction-to-tasklist.md) can be used by humans to complete these tasks. In this last step, you will open Tasklist to run the user task you created.
+When the process instance arrives at the user task, a new user task instance is created at Zeebe. The process instance stops at this point and waits until the user task is completed. Applications like Tasklist can be used by humans to complete these tasks. In this last step, you will open Tasklist to run the user task you created.
 
 :::tip
 While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial. Within this example, we've included a form to demonstrate the completion of a user task.
 
 Using the Zeebe or Tasklist API, many other ways to complete a user task are possible, such as redirecting to another application to complete the task, or even listening to IoT devices to capture human interaction with the real world via job workers.
 :::
 
-<Tabs groupId="install" className="tabs-hidden">
-<TabItem value="saas">
-
-1. Click the **navigation menu icon** next to the Camunda logo in the top bar to open the global navigation, and click **Tasklist** to open the Tasklist application.
-   <img src={ModelerNavImg} style={{width: 200}} alt="Navigation to other applications" />
-
-</TabItem>
-<TabItem value="sm">
-
 1. Open Tasklist at `http://localhost:8080/tasklist`.
 
-</TabItem>
-</Tabs>
-
 2. On the left, you will notice a list of **tasks**. There should be one open task `Decide what's for dinner`. Click this task to open it in the detail view.
 3. In the detail view, the form you created in **[Step 2](#step-2-design-a-form)** appears. It is read only since this task is currently unassigned. You have to claim the task to work on it. Next to **Assignee**, click **Assign to me** to claim the task.
```

Comment:
```suggestion
3. In the detail view, the form you created in **[Step 2](#step-2-design-and-link-a-form)** appears. It is read only since this task is currently unassigned. You have to claim the task to work on it. Next to **Assignee**, click **Assign to me** to claim the task.
```